butter-cms.json

```diff
@@ -1,23 +1,23 @@
 {
     "Timestamp": "2018-08-01T13:38:27.000Z",
     "Name": "ButterCMS",
-    "Version": "06-2019",
+    "Version": "05-2020",
     "License": "Proprietary",
     "Inception": "05-2015",
-    "Category": "Essential",
+    "Category": "Essential - Professional - Enterprise",
     "Open Source": "No",
     "Cloud Service": "Yes",
     "On Premises Installation": "No",
-    "Cloud Service Hosted in Europe": "?",
+    "Cloud Service Hosted in Europe": "No",
     "Commercial Support Available?": "Yes",
-    "GraphQL API": "No",
+    "GraphQL API": "Yes with Gatsby",
```

Comment:
https://buttercms.com/docs/api-client/gatsbyjs

Reply:
I don't think this qualifies. Only native support is meant to be listed in the table.

Reply:
OK.